doc/VMSpecificationSingleNode

```diff
@@ -0,0 +1,15 @@
+### VM Specifications - Single-Node VM deployment
+
+- Please create VM(s) with at least 1 Node, 2 cores (vCPU) and 4GB of RAM.
+- Ensure the VM is created with minimum 6 attached disks (raw) per node i.e. /dev/sdb (Data Disk), /dev/sdc (Metadata Disks)
```

Comment:
`/dev/sdb (Data Disk), /dev/sdc (Metadata Disks)`
This cannot be hard code value.
We just need 6 disks. How to pick them would be a different story.

in my opinion it would be better to fix a hard number to lessen the confusion for the user.

Reply:
hard number you mean by? then how can we mention the same to provide the community

Reply:
and how can we define or mention as Data or Metadata disk otherwise this will confuse the community as which disk to use?

Reply:
Find the devices on each node separately using the command provided below, to fill into respective config.ini sections.
Complete list of attached devices:
`device_list=$(lsblk -nd -o NAME -e 11|grep -v sda|sed 's|sd|/dev/sd|g'|paste -s -d, -)`
Values for storage.cvg.0.metadata_devices:
`echo ${device_list%%,*}`
Values for storage.cvg.0.data_devices:
`echo ${device_list#*,}`

Reply:
thanks verified